### DISCUSSION TITLE
Tracking redirects on the same domain

### DISCUSSION THREAD

**Discussion by nicbou** [plausible/analytics#830] [Anything Else]
On my website, I have many links that redirect to other places. For example, https://allaboutberlin.com/out/appointment-anmeldung. Unfortunately, those are not tracked as outbound links. They are not tracked at all.

Is there a way around this? I'd like them to be included as any other outbound link.

**Comment by ukutaht** [comment]
On top of outbound links, we could add a feature to the script to track clicks on any link with a certain tag, internal or external. Example:

```html
<a href="https://link" plausible-event="Click">My link</a>
```

So this would trigger a `Click` event in plausible when clicked.

Or something like that. I would want to support a similar feature for form submissions as well.
